specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/awsAutoProvisioningSettings.json

```diff
@@ -0,0 +1,247 @@
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/connector/AWS/autoProvisioningSettings": {
+      "get": {
+        "x-ms-examples": {
+          "Get aws auto provisioning settings for subscription": {
+            "$ref": "./examples/AwsAutoProvisioningSettings/GetAwsAutoProvisioningSettingsSubscription_example.json"
+          }
+        },
+        "tags": [
+          "AwsAutoProvisioningSettings"
+        ],
+        "description": "Exposes the aws auto provisioning settings of the subscription.",
+        "operationId": "AwsAutoProvisioningSettings_List",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AwsAutoProvisioningSettingList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/connector/AWS/autoProvisioningSettings/{awsSettingName}": {
+      "get": {
+        "x-ms-examples": {
+          "Get an aws auto provisioning setting for a subscription": {
+            "$ref": "./examples/AwsAutoProvisioningSettings/GetAwsAutoProvisioningSettingSubscription_example.json"
+          }
+        },
+        "tags": [
+          "AwsAutoProvisioningSettings"
+        ],
+        "description": "Details of an aws specific setting.",
+        "operationId": "AwsAutoProvisioningSettings_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/AwsAutoProvisioningSettingName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AwsAutoProvisioningSetting"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
```

Comment:
Why is the connection made at MG level, but settings configured at sub level? That seems random and confusing. Why not embed settings in the connection object, if that is what it is controlling? Which sub is this? If there are dozens in the MG, what criteria is used to choose the sub in this PUT?